src/types/markers.js

```diff
@@ -282,25 +282,31 @@ export type InvalidationPayload = {
  * be included depending on what states happen during the load.  Also note
  * that redirects are logged as well.
  */
+
+type NetworkTimeDetails = {
```

Comment:
@gregtatum I am not sure how the outcome will be here, so let's update this when its time?